api/managementpb/actions.proto

```diff
@@ -93,9 +93,10 @@ message StartMySQLExplainActionRequest {
   string pmm_agent_id = 1;
   // Service ID for this Action.
   string service_id = 2;
-  //
-  // TODO specific parameters if we need them.
-
+  // database for query.
+  string db = 3;
+  // query example.
+  string query = 4;
```

Comment:
I think these params aren't enough. 
There are no credentials (username and password) to prepare dsn.

Reply:
And in StartMySQLJsonExplainActionRequest as well

Reply:
Do you mean we can't get credentials from service and agent?

Reply:
I'm not sure about `db` parameter on UI but it explained in old implementation:
> // If the query has a default db, use it; else, all tables need to be db-qualified
> // or EXPLAIN will throw an error.

Reply:
I mean there is no agent_id param to get credentials from mysqld_exporter and no username and password params to use them. 
are you going to use some logic inside of pmm-managed to get credentials from mysqld_exporter for current service? like `findAgentForService(MYSQLD_EXPORTER, req.ServiceID)`?

Reply:
> are you going to use some logic inside of pmm-managed to get credentials from mysqld_exporter for current service? like findAgentForService(MYSQLD_EXPORTER, req.ServiceID)?
It seems to. We haven't requirements and didn't discuss this. 

@AlekSi @rnovikovP Need your help with that. What should I do?

Reply:
Before GA we should add default username/password to Service itself. They should be used by Agents and Actions when explicit username/password are not given. It is currently tracked there: https://jira.percona.com/browse/PMM-3786, but will be extracted into separate one a bit later.

In the meantime, let's do `findAgentForService(MYSQLD_EXPORTER, req.ServiceID)` with a comment indicating that is a temporary hack.